src/index.html

```diff
@@ -11,7 +11,111 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form
+    action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+    method="post"
+    >
+      <fieldset class="group">
+        <legend>Personal information</legend>
+        <label class="label" for="Surname">
+          Surname:
+          <input class="input" type="text" required>
+        </label>
+        <label class="label" for="Name">
+          Name:
+          <input class="input" type="text" required>
```

Comment:
let's format our code correctly. if there are three or more attributes, you need to put each of them on a new line.
please do this throughout your code

```suggestion
          <input
            class="input"
            type="text"
            required
          >
```